routes/accounts.js

```diff
@@ -80,10 +81,22 @@ function accountRoutes (server, options, next) {
     handler: function (request, reply) {
       var sessionId = toBearerToken(request)
 
-      return accounts.findAll({
-        db: options.db,
-        bearerToken: sessionId,
-        include: request.query.include
+      admins.validateSession(sessionId)
+
+      .catch(function (error) {
+        if (error.toString() === dbErrors.MISSING_DOC.toString()) {
```

Comment:
@gr2m @nolanlawson the errors are [accessible on the PouchDB constructor](https://github.com/pouchdb/pouchdb/blob/master/src/index.js), but currently PouchDB is only a devDependency for hoodie-account-server... can I make it a normal dependency?